server/src/main/java/io/spine/server/aggregate/model/AggregateClass.java

```diff
@@ -69,30 +69,36 @@ protected AggregateClass(Class<A> cls) {
         return result;
     }
 
+    /**
+     * Obtains the set of event classes on which this aggregate class reacts.
+     */
     @Override
-    public final Set<EventClass> eventClasses() {
-        return delegate.eventClasses();
+    public final Set<EventClass> incomingEvents() {
+        return delegate.incomingEvents();
     }
 
+    /**
+     * Obtains the set of <em>external</em> event classes on which this aggregate class reacts.
```

Comment:
As long as we are using `<b>`, let's continue to do that here as well.

Reply:
We use bold for simulating headers. Here is the semantic markup with the emphasis on the word. The way it's rendered is up to styles.